### ISSUE TITLE
during GIT GC, cannot push to server and page /admin/monitor cannot be accessed

### ISSUE THREAD

**Issue by terrywh** [gogs/gogs#4933]
- Gogs version (or commit ref): 0.11.29
- Git version: 2.15.1
- Operating system:  CentOS7
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gogs.io:
  - [ ] Yes (provide example URL)
  - [] No
  - [X] Not relevant
- Log gist (usually found in `log/gogs.log`):

## Description

when run git gc (possiblely with large repos), visit `/admin/monitor` get the following error:
```
template: admin/monitor:52:15: executing "admin/monitor" at <.Pid>: can't evaluate field Pid in type *process.Process
```
also cannot push to gogs server with the following error:
```
$ git push
Gogs: Internal error
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

after gc has finished, everything turn back to normal.

**Comment by Dr-Terrible** [comment]
I got the same error when accessing the admin page at https://localhost:4000/admin/monitor:

> template: admin/monitor:52:15: executing "admin/monitor" at <.Pid>: can't evaluate field Pid in type *process.Process

This message is triggered only when cron jobs are running in the background (i.e.: automatic syncing of the repositories). After the cron jobs are finished, the page is working correctly again.

**Comment by Unknwon** [comment]
Hi, this template error has been fixed in the current latest release 0.11.66.
